scripts/ansible/roles/linux/intel/tasks/sgx-driver.yml

```diff
@@ -8,20 +8,28 @@
 - name: Populate service facts
   service_facts:
 
-- name: Load default driver
-  modprobe:
-    name: intel_sgx
-    state: present
+- name: Include distribution vars
+  include_vars:
+    file: "{{ ansible_distribution | lower }}/main.yml"
+
+- name: Check default driver files
+  stat:
+    path: "{{ item }}"
+  loop: "{{ intel_dcap_driver_files }}"
   ignore_errors: yes
-  register: intel_sgx_module
+  register: intel_sgx_driver
+
+- name:
```

Comment:
nit pick - mind putting a name/description for this stage

Reply:
Thanks for catching this!